autospec/license.py

```diff
@@ -97,13 +97,17 @@ def license_from_copying_hash(copying, srcdir):
         c.setopt(c.FOLLOWLOCATION, 1)
```

Comment:
I recommend setting the `FAILONERROR` option here, so that "fatal" HTTP responses will also raise an exception.

See the recent changes to pkg_integrity.py for an example.